lib/presentation/home_page/components/main_page_weather_content.dart

```diff
@@ -132,10 +134,30 @@ class _MainPageWeatherContentState extends State<MainPageWeatherContent> {
                       ),
                     );
                   } else {
-                    return Center(
-                      child: Text('ładowanie'),
+                    child = Center(
+                      child: Lottie.asset(
+                        'lib/assets/lottie/61302-weather-icon.json',
+                        key: const ValueKey(0),
+                      ),
                     );
                   }
+                  return AnimatedSwitcher(
+                    // switchInCurve: Curves.fastOutSlowIn,
+                    // switchOutCurve: Curves.bounceIn,
+
+                    transitionBuilder: (child, animation) => FadeTransition(
+                      opacity: Tween<double>(begin: 0.0, end: 3.0).animate(
+                        CurvedAnimation(
+                          parent: animation,
+                          curve: Interval(0.7, 1.0),
+                        ),
+                      ),
+                      child: child,
+                    ),
+                    reverseDuration: const Duration(seconds: 18),
```

Comment:
That 18 secs here looks a little bit weird here :) do we need that reverse duration at all?

Reply:
Yep we don't need reverse duration